### PR TITLE
update sceptre-circleci docker image

### PR DESCRIPTION
**Pull request by zaro0508**
Make CI use the latest circle-ci docker image updated to alpine v3.16
with python v3.10 and pyenv pre-installed.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -32,17 +32,12 @@ aliases:
...
       - checkout
       - run:
           name: 'Creating Virtualenv'
-          command: virtualenv venv
-      - restore_cache:
-          key:
-            sceptre-{{ .Environment.CACHE_VERSION }}-requirements-{{ arch }}-{{
-            checksum "requirements/prod.txt" }}-{{ checksum
-            "requirements/dev.txt" }}
+          command: virtualenv --python=/home/circleci/.pyenv/versions/3.8.9/bin/python venv
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply .circleci/config.yml:40]: > Why don't we just use pyenv-virtualenv? That's what we use at my company and it combines pyenv's simplicity with virtual environments. It's pretty straight-forward to set up too. https://github.com/pyenv/pyenv-virtualenv

you are correct, this is just a test to determine the issue.  pyenv-virtualenv isn't pre-installed on circleci images so we may need to install it on the sceptre-circleci image before using it.

[the remark this answers is not in the document]
Comment (zaro0508) [reply .circleci/config.yml:40]: @jfalkenstein https://github.com/Sceptre/sceptre-circleci/pull/15

**Comment by zaro0508** [comment]
it's working with pyenv now @jfalkenstein

**Review by tarkatronic — APPROVED** [review]
It will be so nice when this eventually moves over to GitHub actions... you don't have to mess with any of this stuff then. 😄

**Pull request merged**